app/src/main/java/com/main/justthetip/MainActivity.java

```diff
@@ -1,13 +1,59 @@
 package com.main.justthetip;
```

Comment:
You guys can leave comments right in the code if you wanna ask questions or anything.